src/lib/xiph/meson.build

```diff
@@ -1,6 +1,6 @@
 libflac_dep = dependency('flac', version: '>= 1.2', required: get_option('flac'))
 
-if is_windows
+if is_windows and libflac_dep.found()
   # Our Windows build generates a static libFLAC build
   libflac_dep = declare_dependency(compile_args: '-DFLAC__NO_DLL',
```

Comment:
Another way to fix is to make libflac_dep a disabler. OTOH, this is simpler.

Reply:
Disablers are nice, but not applicable here, because this would simply disable the whole MPD executable.